webapp/components/ContributionForm/ContributionForm.spec.js

```diff
@@ -384,6 +384,15 @@ describe('ContributionForm.vue', () => {
           await wrapper.find('form').trigger('submit')
           expect(mocks.$apollo.mutate).toHaveBeenCalledWith(expect.objectContaining(expectedParams))
         })
+
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2020-01-16T14:40:09Z" title="Thursday, January 16th 2020, 3:40:09 pm +01:00">Jan 16, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -235,6 +235,12 @@ describe('ContributionForm.vue', () => {
           expect(mocks.$apollo.mutate).toHaveBeenCalledWith(expect.objectContaining(expectedParams))
         })
 
+        it('deleted a teaser image', async () => {
```


```suggestion
        it('deletes a teaser image', async () => {
```